src/bloqade/builder/coupling.py

```diff
@@ -1,108 +1,38 @@
-            * [Two Qubit Adiabatic Sweep](https://queracomputing.github.io/bloqade-python-examples/latest/examples/example-2-two-qubit-adiabatic/)
-            * [1D Z2 State Preparation](https://queracomputing.github.io/bloqade-python-examples/latest/examples/example-3-time-sweep/)
-            * [2D State Preparation](https://queracomputing.github.io/bloqade-python-examples/latest/examples/example-3-2d-ordered-state/)
-            * [Quantum Scar Dynamics](https://queracomputing.github.io/bloqade-python-examples/latest/examples/example-4-quantum-scar-dynamics/)
-            * [Solving the Maximal Independent Set Problem on defective King Graph](https://queracomputing.github.io/bloqade-python-examples/latest/examples/example-5-MIS-UDG/)
-
-        ??? warning "Potential Pitfalls"
-
-            Bloqade allows you to build a field for the Detuning in the form of:
-
-            $$
-            \sum_j \Delta_j(t)  = \sum_j \sum_a C^{a}_{j} f_{a}(t)
-            $$
-
-            Where your field can contain multiple drives.
-
-            In reality the hardware only supports the following configuration:
-
-            $$
-            \Delta_{i}(t) = \Delta_{1}(t) + c_{i} \Delta_{2}(t)
-            $$
-
-            $$
-            c_i \in [0, 1]
-            $$
-
-            $$
-            \Delta_{2}(t) \leq 0
-            $$
-
-            Where $\Delta_{1}(t)$ is your global detuning (establishable via [`uniform`][bloqade.builder.field.Detuning.uniform]) and $\Delta_{2}(t)$ is your
-            local detuning waveform with the spatial modulation $c_{i}$ establishable via [`location`][bloqade.builder.field.Detuning.location] or [`scale`][bloqade.builder.field.Detuning.scale].
-
-        # Next Possible Steps
-
-        You may continue building your program via:
-
-        - [`uniform`][bloqade.builder.field.Detuning.uniform]: To address all atoms in the field
-        - [`location(locations, scales)`][bloqade.builder.field.Detuning.location]: To address atoms at specific
-            locations via indices
-        - [`scale(coeffs)`][bloqade.builder.field.Detuning.scale]: To address all atoms with an individual scale factor
+        Raises:
+            None
 
+        Note:
+            The detuning specifies how off-resonant the laser being applied to the atoms is from the atomic energy transition, driven by the Rabi frequency.
+
+            Example:
+                ```python
+                from bloqade import start
+                geometry = start.add_position((0,0))
+                coupling = geometry.rydberg
+                coupling.detuning
+                ```
+
+            - Next Possible Steps
+            You may continue building your program via:
+            - `uniform`: To address all atoms in the field
+            - `location(locations, scales)`: To address atoms at specific locations via indices
+            - `scale(coeffs)`: To address all atoms with an individual scale factor
```

Comment:
Could you try to implement cross references here? In the removed docstring it would look something like this:

```
        - [`uniform`][bloqade.builder.field.Detuning.uniform]: To address all atoms in the field
        - [`location(locations, scales)`][bloqade.builder.field.Detuning.location]: To address atoms at specific
            locations via indices
        - [`scale(coeffs)`][bloqade.builder.field.Detuning.scale]: To address all atoms with an individual scale factor
```
And when you build the docs they come up as links that you can click on and jump to the other respective docstrings